src/ssb_project_cli/ssb_project/app.py

```diff
@@ -661,6 +663,55 @@ def clean(
     print(f"Deleted Jupyter kernel {project_name}.")
 
 
+def clean_virtual_env(project_name: str) -> None:
+    """Removes virtual environment associated to a specific project name.
+
+    Args:
+        project_name: The project the virtual environment is associated to.
+    """
+    typer.echo(f"Removing virtual environment for {project_name}...")
+
+    find_envs_cmd = f"ls -d /home/jovyan/.cache/pypoetry/virtualenvs/{project_name}*"
```

Comment:
I don't think it's the best solution to have the path hardcoded, it won't work outside Jupyter environments, or if this path changes. We could use the command `poetry config virtualenvs.path` to retrieve the correct path at runtime.

Reply:
was looking for something like this, thanks!